asn1crypto/cms.py

```diff
@@ -101,6 +102,13 @@ class CMSAttributeType(ObjectIdentifier):
         '1.2.840.113549.1.9.16.2.14': 'signature_time_stamp_token',
         # https://tools.ietf.org/html/rfc6211#page-5
         '1.2.840.113549.1.9.52': 'cms_algorithm_protection',
+        # http://www.cisco.com/c/en/us/support/docs/security-vpn/public-key-infrastructure-pki/116167-technote-scep-00.html
```

Comment:
This should reference `https://tools.ietf.org/html/draft-gutmann-scep-06#section-3.2.1`